### DISCUSSION TITLE
插件发布后，安装时提示包无效。

### DISCUSSION THREAD

**Discussion by peachtao2024** [microsoft/MicrosoftEdge-Extensions#192] [Q&A]
https://microsoftedge.microsoft.com/addons/detail/dnmhncgoahhemfleijncbcdnlcnhhddh
extension发布后，安装时提示“包无效”。
经过排查，上传的zip文件正常，用户安装时crx的目录层级丢失，**文件路径images/icon_16.png变为了images_icon_16.png(文件路径变为了下划线)**

正常的目录结构
![image](https://github.com/user-attachments/assets/8f7a2056-c91d-49d8-90cd-f5330227c33d)

下载的crx目录结构
![image](https://github.com/user-attachments/assets/2d3623bf-8dbc-4e38-99f8-90a9feb52c65)

我不清楚edge将zip转换为crx的时候，是不是出了什么bug，希望帮忙解决下。

tks

**Comment by peachtao2024** [comment]
插件下载地址：
https://msedgeextensions.sf.tlu.dl.delivery.mp.microsoft.com/filestreamingservice/files/bcc8a0a2-8791-4a48-bf70-41763ea91e07?P1=1726554973&P2=404&P3=2&P4=iKqaAvcdgVogZ%2fVQz14dYpV9ZL2fBb1qOgmMhbgnDDSCHTm5Z3NfAYWaFGY8kvg6N7MnVKLTPWimuqyppA5fDg%3d%3d

**Comment by ManikanthMSFT** [comment]
Hi @peachtao2024, Thank you for reporting this. We are looking into this and will let you know as soon as we have an update.

**Reply by peachtao2024** [reply]
已经解决了，tks

**Reply by peachtao2024** [reply]
是winrar和7zip对于压缩包的处理不一致导致的该问题，我使用7zip进行打包后，问题就解决了。tks

**Reply by ManikanthMSFT** [reply]
Hey @peachtao2024, thank you for confirming that your concern has been resolved. If you have any other questions or need further assistance, please don’t hesitate to reach out.
